cyder/management/commands/dhcp_migrate.py

```diff
@@ -408,6 +408,7 @@ def migrate_dynamic_hosts():
                     'address {}\n'.format(intr.mac))
                 stderr.write('    {}\n'.format(e))
                 intr = None
+                s.delete()
```

Comment:
This isn't a mistake, the system gets deleted because the associated interface was unable to be created. We don't want any orphaned systems from migrations.

commit: 81071308d7f1f138deae9f7ed9c47f46208558b4